src/commands/flinkUDFs.ts

```diff
@@ -17,12 +18,8 @@ import { UriMetadata } from "../storage/types";
 import { FlinkDatabaseViewProvider } from "../viewProviders/flinkDatabase";
 import { FlinkDatabaseViewProviderMode } from "../viewProviders/multiViewDelegates/constants";
 import { promptForFunctionAndClassName } from "./utils/uploadArtifactOrUDF";
-const logger = new Logger("flinkUDFs");
 
-export async function setFlinkUDFViewModeCommand() {
```

Comment:
Just moved under `registerFlinkUDFCommands()` so the command registering is at the top of the file and not the middle